doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,161 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Group                                                   | The whole space is equally divided into groups with fixed size. That means every group has the same amount of blocks. When allocating spaces, groups are iterated to search for the best candidate. Group is locked during this step. Having multiple groups in a container can reduce the lock contention.                                                                                                                                                                                                                                                                            |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| HA interface                                            | HA interface API that allows Halon to control Motr and allows Motr to receive cluster state information from Halon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
s/Halon/Hare

But you need to add two more things:
HA
HARE

Reply:
Added.